elastic/kibana
# [Security Solution][Entity Analytics] Updates to Risk Scoring Explanation Flyout

## Summary

This PR updates the "Entity Risk Analytics" flyout that opens when you click the "How is the risk score calculated?" link on the Entity Store page. A PR instance can be found [here](https://rylnd-pr-176358-8490-risk-explanation-updates.kbndev.co/app/security/entity_analytics_management)

### Changes
* Updates existing copy with some minor phrasing changes
* Adds new copy, including a new `<ol>` of calculation steps
* Adds an additional table for Asset Criticality Tiers
* Moves our "Asset Criticality Tier to Risk Weight" mapping to a shared location, so that it can be used to build the above table

### How to review
1. Navigate to the "Entity Risk Score" page (Security -> Manage -> Entity Risk Score)
2. Click the "How is the risk score calculated?" link to open the flyout
3. Observe the updated flyout
4. (optional) Appreciate the flyout's majesty

### Screenshots
<img width="643" alt="Screenshot 2024-02-13 at 2 55 32 PM" src="https://github.com/elastic/kibana/assets/657252/5b5dff46-b98a-451b-849b-678088638187">
<img width="643" alt="Screenshot 2024-02-13 at 2 55 49 PM" src="https://github.com/elastic/kibana/assets/657252/3271023a-f94e-45a9-b32b-71ce1f6d9e83">


### Checklist

Delete any items that are not applicable to this PR.



### For maintainers
